sites/public/src/pages/sign-in.tsx

```diff
@@ -119,31 +121,84 @@ const SignIn = () => {
 
   useEffect(() => {
     if (
-      networkError?.error.response.data?.message ===
+      networkError?.error?.response?.data?.message ===
       EnumUserErrorExtraModelUserErrorMessages.accountNotConfirmed
     ) {
       setConfirmationStatusModal(true)
     }
   }, [networkError])
 
+  const MobileComponent = () => {
+    return (
+      <HeadingGroup
+        heading={t("account.signUpSaveTime.title")}
+        subheading={t("account.signUpSaveTime.subTitle")}
+        size="xl"
+        className="p-4 -order-1"
+      />
+    )
+  }
+
+  const DesktopComponent = () => (
+    <HeadingGroup
+      heading={t("account.signUpSaveTime.title")}
+      subheading={t("account.signUpSaveTime.subTitle")}
+      size="xl"
+      className="grow-0 shrink-1"
+    />
+  )
+
   return (
     <>
-      <FormsLayout>
-        <FormSignIn
-          onSubmit={(data) => void onSubmit(data)}
-          control={{ register, errors, handleSubmit, watch }}
-          networkStatus={{
-            content: networkStatusContent,
-            type: networkStatusType,
-            reset: () => {
-              reset()
-              resetNetworkError()
-              setConfirmationStatusMessage(undefined)
-            },
-          }}
-          showRegisterBtn={true}
-        />
-      </FormsLayout>
+      {process.env.showMandatedAccounts ? (
+        <FormsLayout className="sm:max-w-lg md:max-w-full">
+          <div className="flex flex-col md:flex-row md:ml-20 justify-center">
+            <div className="display md:hidden ">
+              <MobileComponent />
+            </div>
+            <div className="md:max-w-lg w-full justify-center">
+              <FormSignIn
```

Comment:
This block is repeated below, can we define it above once?

Reply:
I had some odd keyboard focus issues as a result of doing something similar before. I can take another look. I do like the env var toggle simplicity, but I could also pull the classNames into a module and toggle those based on env var, though the DOM structure for the default view may have a extra div elements

Reply:
Yeah I ran into those keyboard issues again. Leaving for now, open for ideas!

Reply:
@emilyjablonski @ColinBuyck Just clarifying that I plan on leaving this as is for now. Let me know your thoughts!

Reply:
Would you be able to screen record or describe the keyboard focus issues more? Curious to see if its something we've encountered before and may have a known solution for

Reply:
Same, would love to pair!

Reply:
<img width="750" alt="Screenshot 2024-01-30 at 10 01 46 AM" src="https://github.com/bloom-housing/bloom/assets/131277283/534eb540-7697-4530-9d9e-6f7996ba57c3">

https://github.com/bloom-housing/bloom/assets/131277283/682b1e28-bd75-4bfd-9cd5-8f06cbe45300

Here is what I am seeing when I pull the SignIn code into a component. Let me know if I am missing something!